### PR TITLE
Feature/cta button

### PR DESCRIPTION
**Pull request by jahlela**
- Replace the existing newsletter banner with a call to action. 
- Add newsletter button to footer.

Screen Shot 2019-08-09 at 11 54 07 AM

Screen Shot 2019-08-12 at 12 41 34 PM

### CODE REVIEW

**Comment by bbinoy** [comment]
People should still be able to sign up for the newsletter. Where can they do that now? Isn't it better to have 2 CTA: Signup OR subscribe?

I know you (Spela) thought about it, but I think from a user perspective 50MB storage is not the biggest selling point. What about "Free forever for your cap table"?

File: src/layouts/utils.jsx [hunk 1]
```diff
@@ -87,3 +89,25 @@ export const animateTablet = () => {
...
+  {
+    name: 'peaceOfMind',
+    title: <Trans>Find peace of mind with a better cap table tool</Trans>,
+    CTA: <Trans>Get started</Trans>
+  }
+];
+
+// eslint-disable-next-line import/prefer-default-export
+export const CTAExperiment =
+  typeof window !== 'undefined' ? sample(CTAExperiments) : CTAExperiments[0];
```
Comment (jahlela) [inline src/layouts/utils.jsx]: Add last experiment:

```
"Your cap table, free forever"
"Ton cap table, gratuit pour toujours"
"Dein Cap Table, für immer kostenlos"
```

File: src/layouts/index.jsx [hunk 2]
```diff
@@ -83,218 +89,248 @@ const Nav = (props: LayoutProps) => (
...
-              <Trans>Career</Trans>
-            </Link>
-            <Link className="nav-link" href to={`${props.prefix}/contact/`}>
-              <Trans>Contact & Imprint</Trans>
-            </Link>
-          </div>
+          <a
+            className="btn btn-lg btn-primary mx-5 ml-md-5 mr-md-0"
+            href={`${appUrl}/signup`}
+            onClick={() => trackSignup('clickSignup')}
```
Comment (jahlela) [inline src/layouts/index.jsx]: Move experiment trigger to this `onClick` event, and remove `useEffect`

File: src/layouts/utils.jsx (hunk 1 above)
Comment (morloy) [reply src/layouts/utils.jsx]: If the experiments are not used in the session I wouldn’t store them in `window`. Just sample them on each component render.

Comment (morloy) [reply src/layouts/utils.jsx]: The `CTA` field seems unnecessary here, it’s the same for all experiments.

Comment (morloy) [reply src/layouts/utils.jsx]: Abbreviations like `CTA` are not nice for readability. (Unless everyone knows what a CTA is.) I would just call it `callToAction` or `signupBanner`.

File: src/layouts/index.jsx [hunk 3]
```diff
@@ -83,218 +89,248 @@ const Nav = (props: LayoutProps) => (
...
-                    🍔
-                  </span>
-                  English
-                </Link>
-                <Link
-                  className="dropdown-item d-flex"
-                  to={`/de${deprefix(props.location.pathname)}`}
-                  href
+                <FontAwesomeIcon className="newsletter-icon" icon={faEnvelope} title="Newsletter" />{' '}
+                Newsletter
```
Comment (morloy) [inline src/layouts/index.jsx]:
```suggestion
                <Trans>Newsletter</Trans>
```

File: src/layouts/index.jsx [hunk 4]
```diff
@@ -83,218 +89,248 @@ const Nav = (props: LayoutProps) => (
...
-                  <span className="mr-1" role="img" aria-label="English">
-                    🍔
-                  </span>
-                  English
-                </Link>
-                <Link
-                  className="dropdown-item d-flex"
-                  to={`/de${deprefix(props.location.pathname)}`}
-                  href
+                <FontAwesomeIcon className="newsletter-icon" icon={faEnvelope} title="Newsletter" />{' '}
```
Comment (morloy) [inline src/layouts/index.jsx]:
```suggestion
                <FontAwesomeIcon className="newsletter-icon mr-2" icon={faEnvelope} title="Newsletter" />
```

File: src/layouts/index.jsx [hunk 5]
```diff
@@ -12,23 +12,29 @@ import {
...
 } from '@fortawesome/free-brands-svg-icons';
+import { faEnvelope } from '@fortawesome/free-solid-svg-icons';
 import 'typeface-roboto-slab'; // eslint-disable-line import/extensions
 import 'typeface-work-sans'; // eslint-disable-line import/extensions
 import 'katex/dist/katex.min.css';
 import 'prism-themes/themes/prism-ghcolors.css';
 
-import { Title, name, appUrl, loadScript, targetBlank, animateTablet, trackSignup } from './utils';
-import { catalogs, langFromPath, langPrefix, getLocale, deprefix } from '../i18n-config';
-import NewsletterForm from '../components/NewsletterForm';
```
Comment (morloy) [inline src/layouts/index.jsx]: Don’t forget to remove the file as well.

File: src/layouts/style.scss [hunk 6]
```diff
@@ -322,6 +341,24 @@ h1 {
...
+    p {
+      font-size: 22px;
+      width: 100%;
+      text-align: center;
+    }
+
+    a {
+      width: 100%;
+    }
+  }
```
Comment (morloy) [inline src/layouts/style.scss]: It seems like most of the styling here can be done with Bootstrap classes. It’s better to reuse them and not add custom CSS. You can ask Uri or me if you need help with this.

File: src/layouts/index.jsx [hunk 7]
```diff
@@ -83,218 +89,248 @@ const Nav = (props: LayoutProps) => (
...
-              Convertible Loans
-            </Link>
-            <Link className="nav-link" href to={`${props.prefix}/updates/kpis-and-reports/`}>
-              KPIs & Reports
-            </Link>
+const Footer = (props: LayoutProps) => {
+  useEffect(() => {
+    window.CTAExperiment = CTAExperiment.name;
+    if (window.ga) window.ga('set', 'dimension1', window.CTAExperiment);
+  }, []);
```
Comment (morloy) [inline src/layouts/index.jsx]: This is duplicated in `CTABanner` and should be removed as well.

File: src/layouts/utils.jsx (hunk 1 above)
Comment (jahlela) [reply src/layouts/utils.jsx]: I don't see how these are stored in `window`? It's just checking that the app's running in the browser, and if not, just pulls the first experiment option. You can see the same pattern here: https://github.com/morloy/ledgy.com/blob/f7c28395eb26269ecbcbd64c129f2ee622ce4cef/src/pages/index.jsx#L49

There are future plans for different CTAs, which is why it's here as a field, even though they're all the same. 

Sure, CTA might not be as widely known. But it beats a one-letter variable that could almost literally be anything ;) I'll rename it.

File: src/layouts/index.jsx [hunk 8]
```diff
@@ -83,136 +90,165 @@ const Nav = (props: LayoutProps) => (
...
+            </Link>
+            <Link className="nav-link" href to={`${props.prefix}/legal/cookie-policy/`}>
+              <Trans>Cookie Policy</Trans>
+            </Link>
+            <Link className="nav-link" href to={`${props.prefix}/legal/gdpr/`}>
+              <Trans>GDPR</Trans>
+            </Link>
+          </div>
+        </div>
+        <div className="col-12 col-md-2 pl-6 order-md-1">
```
Comment (morloy) [inline src/layouts/index.jsx]: You’re reverting Uri’s new footer structure in this PR 😉

Comment (jahlela) [reply src/layouts/index.jsx]: No longer!

**Pull request merged**